packages/charts/package-lock.json

```diff
@@ -1,64 +1,25 @@
 {
```

Comment:
this file probably shouldn't exist... it gets added when you do `npm i something` inside the package... remove it and do an install at the top level to get the correct hoisting